array-manipulation-2/src/drop-right.js

```diff
@@ -0,0 +1,10 @@
+/* exported dropRight */
+function dropRight(array, count) {
+  var newArray = [];
+  for (var i = 0; i < array.length - count; i++) {
+    if (count > array.length) {
+      newArray = array;
```

Comment:
This `if` statement can come before your loop and just `return` the `array` immediately.